### ISSUE TITLE
Merged schema without Query, Mutation and Subscription

### ISSUE THREAD

**Issue by Fi1osof** [Urigo/merge-graphql-schemas#155]
```
let user = `type User {
id: ID
}`;

let extended = `type User {
username: String
}`;

return  mergeTypes([user, extended], {all: true});
```

Result:
```
schema {
  query: Query
}

type User {
  id: ID
  username: String
}

```

Need only : 
```
type User {
  id: ID
  username: String
}
```

**Comment by RichardLitt** [comment]
Thanks @Fi1osof. I'm a bit confused what you are suggesting here; is this a bug? Are you requesting a feature? Thanks.

**Comment by Fi1osof** [comment]
@RichardLitt , sorry, forgot type this is feature, not bug. But here schema builds with Query, Mutation and Subscription, event when them does not exists: https://github.com/okgrow/merge-graphql-schemas/blob/master/src/merge_types.js#L108-L110

Maybe this helps understand what i need: 
```
const { parse, print } = require('graphql');
...
const parsed = parse(typesMerged);

parsed.definitions = parsed.definitions.filter(
  n => n.kind !== "SchemaDefinition" && !(
    n.kind === "ObjectTypeDefinition" && ["Query", "Mutation", "Subscription"].indexOf(n.name.value) !== -1)
  );

let schema = print(parsed);
```
Here schema without Query and etc., types, enums and etc only.

**Comment by RichardLitt** [comment]
Thanks for clarifying! I'm going to have one of the other maintainers chime in on this; I'm not sure I understand the problem well enough to help offer advice here, but thanks for making the question clearer.

**Comment by Fi1osof** [comment]
@RichardLitt it needs for https://github.com/prisma/prisma-binding
For base schema not define Query and other. Then prisma generate complete schema. When merged schema contains Query/Mutation/Subscription prisma create tables in database for them.

**Comment by kyawkyawsoezhu** [comment]
@Fi1osof  how do you manage to remove

```
schema {
  query: Query
}
```

**Comment by kamilkisiela** [comment]
#196 introduces `schemaDefinition` option which enables that

**Comment by kamilkisiela** [comment]
It's available now under `v1.6.0-beta.2`
